aten/src/ATen/native/Normalization.cpp

```diff
@@ -485,6 +485,23 @@ std::tuple<Tensor, Tensor, Tensor> batch_norm_backward_cpu_template(
 std::tuple<Tensor, Tensor, Tensor, Tensor, int64_t> _batch_norm_impl_index(
```

Comment:
Moving forward, this won't be called after batch norm consolidation. I would recommend moving this check to the `batch_norm` function itself